client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -139,77 +129,6 @@ type PlanFeatures2023GridState = {
 	showPlansComparisonGrid: boolean;
 };
 
-const PlanLogo: React.FunctionComponent< {
```

Comment:
As mentioned above, this has been moved to its own file for handling component state handling.